Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1161,7 +1161,9 @@ public class BrowserViewController: UIViewController {
 
     header.snp.remakeConstraints { make in
       if self.isUsingBottomBar {
-        if let keyboardHeight = keyboardState?.intersectionHeightForView(self.view), keyboardHeight > 0, presentedViewController == nil {
+        // Need to check Find In Page Bar is enabled in order to aligh it properly when bottom-bar is enabled
+        if let keyboardHeight = keyboardState?.intersectionHeightForView(self.view), keyboardHeight > 0,
+           (presentedViewController == nil || findInPageBar != nil) {
```

Comment:
This is added for Find-In Page presented under the keyboard when Bottom Bar is selected.

Reply:
We will want this change specifically in 1.44 FYI

Reply:
I didn't think about that. @iccub should we move everything to 1.44 or only this part separately ?

Reply:
easier to have both imo

Reply:
Ok changing milestone of the ticket